app/views/layouts/admin.html.erb

```diff
@@ -22,7 +22,7 @@
       </div>
     </nav>
 
-  <p class="flash"><%= flash.notice %></p>
+    <p class="flash"><%= flash.alert %></p>
```

Comment:
Cater for both alert and notice using bootstrap alerts for each case (red for alert, green for notice)